### ISSUE TITLE
Library / Grid view redesign

### ISSUE THREAD

**Issue by RobbWatershed** [avluis/Hentoid#988]
- [x] Experiment with alternate design for grid view cards
  - Minimal setup : Cover only
  - ~~Hentai2Read-style~~
  - ~~Commands on dropdown (submarine) menu~~

- [x] ~~Option for~~ Bezel-less/frameless grid between books

- [x] Hideable (toggle-able) heart, source and rating icons

- [x] Option to choose ~~the number of columns and/or~~ the preview cover size (see #713 for an example)

- [x] Selection border cosmetics
  - ~~Fixed card size to avoid size discrepancies when selecting books~~ => Can't repro
  - Prevent selection border clipping behind book cover => DONE

**Comment by RobbWatershed** [comment]
See #948  #750 and #713 too

**Comment by RobbWatershed** [comment]
The base problem is that the current UI tries to push all essential information to everyone, whereas many users want a minimalist UI and consider the grid tile layout to be bloated.

We listed three layout alternatives :
I don't believe the H2R-style layout is a viable solution as it requires two taps to read the book, which kinda kills the main point of the library screen

Putting the commands (set fav / set rating / view source) on a dropdown menu goes against Material design guidelines and our recent decommissioning of dropdown menus. Plus people can rate and fav books using 1/ the reader and 2/ multiple selection on the main library screen, and thus don't need these features accessible from a dropdown menu

=> I decided the best way to go is to make the grid tile display entirely modular to allow users to see what they actually need, the default being "every UI element is visible" 

***

These options have no effect on the "list" view mode as it should be the fallback for grid mode users who occasionally need to access detailed information (e.g. when sorting their books according to their size)
